packages/react/src/index.tsx

```diff
@@ -0,0 +1,20 @@
+import './polyfills';
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+
+const root = ReactDOM.createRoot(
+  document.getElementById('root') as HTMLElement
+);
+root.render(
+  // <React.StrictMode> // T1094699
```

Comment:
The bug already fixed. Maybe this block of code can be uncommented?

Reply:
I checked the fix hasn't yet been included in any of the current release versions